src/test/java/com/sonyericsson/jenkins/plugins/bfa/BuildFailureScannerHudsonTest.java

```diff
@@ -609,6 +610,120 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
     }
 
     /**
+     * Test enableBuildDescription = false does not append failure cause to build description
+     */
+    @Test
+    public void testTestEnableBuildDescriptionIfDisabled() throws Exception {
+      PluginImpl.getInstance().setTestResultParsingEnabled(true);
```

Comment:
what does test results have to do with this feature?

Reply:
They allow a build that fails with some failure causes.  It allows the test to make sure the build description is not modified when the feature is disabled.